### PR TITLE
Optimize metrics

### PR DESCRIPTION
**Pull request by shipko**
We actively using defect dojo for collecting findings.

On the current moment we have 10 000 findings. 

We noticed that when we loading a metrics pages our god is falling down with 500 error (out of memory problem). To solve the problem we optimized a metric page. This PR is contain solve for optimization 

This improvement halves the memory consumption for finding processing

### CODE REVIEW

File: dojo/templates/dojo/metrics.html [hunk 1]
```diff
@@ -114,42 +114,44 @@
...
+                <div class="panel-heading tight">
+                    <div class="clearfix">
+                        <h3 {% if not critical_prods %}class="has-filters" {% endif %}>
+                            {{ name }}
+                            {% comment %} {% if not critical_prods %}
+                                <div class="dropdown pull-right"></div>
+                            {% endif %} {% endcomment %}
+                            {% if not critical_prods %}
+                                <div class="dropdown pull-right">
+                                    <button id="show-filters" data-toggle="collapse" data-target="#the-filters" class="btn btn-primary toggle-filters"> <i class="fa fa-filter"></i> <i class="caret"></i> </button>
```
[the remark this answers is not in the document]
Comment (shipko) [reply dojo/templates/dojo/metrics.html:128]: If accesslint talks about #the-filters - it'wrong

**Comment by shipko** [comment]
@Maffooch @damiencarol please check k8s deployment test. In some cases test fail without a reason

And can you restart failed test?

**Comment by mtesauro** [comment]
@shipko Restarted the failed tests...

And, yes, they haven't been very reliable of late.

**Review by Maffooch — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Pull request merged**